branchff

```diff
@@ -92,13 +90,19 @@ gitlib::repo_state || common::exit 1
 ###############################################################################
 # MAIN
 ###############################################################################
-BASEDIR="/usr/local/google/$USER"
+BASEDIR="$(mktemp -d "$TMPDIR/$PROG.XXXXXX")"
 # WORK/BUILD area
 WORKDIR=$BASEDIR/$PROG-$RELEASE_BRANCH
 # Go tools expect the kubernetes src to be under $GOPATH
 export GOPATH=$WORKDIR
 # TREE_ROOT is working branch/tree
 TREE_ROOT=$WORKDIR/src/k8s.io/kubernetes
+
+# Set a trap to cleanup BASEDIR if CLEANUP="yes" is set.
```

Comment:
Is this a new env? Why not another "flag"?

Reply:
Good point! It was mainly because I had no idea how we were generating the flags before, but I found it here: https://github.com/kubernetes/release/blob/c32c094ff025d4d61fc27e575b489a174090c630/lib/common.sh#L531-L565

Fixed up in the most recent push.